packages/contracts/contracts/examples/NativeHashi721Example.sol

```diff
@@ -35,8 +35,4 @@ contract NativeHashi721Example is NativeHashi721 {
   function _baseURI() internal view virtual override returns (string memory) {
     return _baseTokenURI;
   }
-
-  function tokenURI(uint256 tokenId) public view override returns (string memory) {
```

Comment:
I think you should not remove this method

Reply:
If you remove this, this NFT metadata can not be retrieved by 3rd party services.